test/evergreen/run_model_workloads.sh

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+#
+# Run the test/model workloads that failed previously.
+
+# Switch to the model-test directory.
+cd $(git rev-parse --show-toplevel)/cmake_build/test/model/tools || exit 1
```

Comment:
It would be preferable to have Evergreen set the working directory, like how test/checkpoint etc do it. You can do this with the `working_dir` directive, e.g. `working_dir: "wiredtiger/cmake_build/test/checkpoint"`.

I find this preferable so that it's easier to run tests locally - the first thing I try is `./script.sh` and this will immediately break. It's also the case that not all build directories are called `cmake_build`.

Reply:
That's an interesting point! I didn't think about this issue, as I just paralleled the test after `run_format_configs.sh`. Just like in the case of format, it uses Evergreen's `working_dir` directive to switch to where the script is located, which is `wiredtiger/test/evergreen` in both cases, and then they both `cd` to the appropriate build directories.

One way to address the issue would be to move the script from `wiredtiger/test/evergreen` to somewhere under `wiredtiger/test/model`. The script could then use relative paths to find both the binary and the workload files (which would presumably not be copied to the build directory).

Or is there any advantage to keeping the script in `wiredtiger/test/evergreen` like `run_format_configs.sh` (even though we would then have to keep the build path in the script)?

Reply:
Ah OK. If there's precedent for doing this, then I can live with it.

I would slightly prefer to have the script alongside the model and use relative paths for the workload files, but if the amount of work doesn't justify the upside then I'm OK to leave it as-is.